app/presenters/concerns/hyrax/iiif_av/displays_content.rb

```diff
@@ -102,8 +104,16 @@ def video_display_content(url, label = '')
         def audio_content
           if solr_document['derivatives_metadata_ssi'].present?
             files_metadata = JSON.parse(solr_document['derivatives_metadata_ssi'])
-            external_files = files_metadata.select { |f| f['external_file_uri'].present? }
-            return external_files.map { |f| audio_display_content(f['external_file_uri'], f['label']) } unless external_files.empty?
+            external_files = files_metadata.select { |f| f['file_location_uri'].present? }
```

Comment:
`audio_content` and `video_content` look like they can be DRYed up significantly.

Reply:
Now that these have grown I think it makes sense.